tests/test_simple.py

```diff
@@ -252,6 +252,15 @@ def test_single_image():
     test_params = psf.interp.interpolate(target)
     numpy.testing.assert_almost_equal(test_params, true_params, decimal=5)
 
+    # Test that we can make rho statistics
+    stats = piff.RhoStatistics.read(psf_file)
+    logr, rho1, rho2, rho3, rho4, rho5 = stats.stats(stars, logger=logger, max_sep=5000)
```

Comment:
Should do some basic tests of the result here.
- check the range of separations returned
- check that the max absolute value of each rho stat is less than some value
- check that they aren't precisely zero.